python/cuspatial/cuspatial/utils/join_utils.py

```diff
@@ -31,7 +32,7 @@ def apply_binarize(in_col, width):
     if out.size > 0:
```

Comment:
Would it make sense to simply use a CuPy array here?

```suggestion
    out = cupy.asarray(buf).view("int8").reshape((in_col.size, width))
    if out.size > 0:
```

Reply:
Didn't even realize we had JIT Numba kernels in cuSpatial. Is there not a native function from libcuspatial or libcudf we can use?

Reply:
> Didn't even realize we had JIT Numba kernels in cuSpatial. Is there not a native function from libcuspatial or libcudf we can use?

I think this kernel is to convert the result of point in polygon primitive, which presents the all pairs result in a compact form (32 bits) in to (N, 32) boolean arrays. Libcuspatial API is designed to minimize memory consumption, but cuspatial python present the API so that it's compatible to dataframe protocols. At that time it seems this conversion helper should live in python not the C++ layer.

Reply:
> Would it make sense to simply use a CuPy array here?

Done.

Reply:
Looks good. Thanks Michael! 🙏